docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -45,12 +46,282 @@ For information about how to support multiple tenants, see xref:security-openid-
 
 == Using the authorization code flow mechanism
 
+=== Configuring access to the OIDC Provider endpoint
```

Comment:
Use sentence-case for titles, so only first word and proper nouns are capitalized.

```suggestion
=== Configuring access to the OIDC provider endpoint
```